README.md

```diff
@@ -544,91 +575,86 @@ PayCart payCart = PayCart.builder()
   .build();
 ```
 
-Additionally `PayCart` provides two functions to create and prepare 
+Additionally, `PayCart` provides two functions to create and prepare:
```

Comment:
not sure about "to create and prepare" -- can we clarify that further? (what's being created?)